pkg/cmd/server/kubernetes/network/network.go

```diff
@@ -92,7 +92,11 @@ func (c *NetworkConfig) RunProxy() {
 	case componentconfig.ProxyModeIPTables:
 		glog.V(0).Info("Using iptables Proxier.")
 		if bindAddr.Equal(net.IPv4zero) {
-			bindAddr = getNodeIP(c.ExternalKubeClientset.CoreV1(), hostname)
+			var err error
+			bindAddr, err = getNodeIP(c.ExternalKubeClientset.CoreV1(), hostname)
```

Comment:
This change is in iptables proxy and not userspace proxy, description talks about userspace proxy issue? (this could be lack of my understanding)

Reply:
No, that's a good point.  I am being imprecise.  What happens is that the iptables proxier needs to know the ip address, but it only matters when the userspace proxy is being used.  The iptables points to that ip when it sends traffic to the port for the userspace proxy.  So it doesn't show up in the testing as a problem until the unidler is used.

Reply:
Okay understood, thanks for the explanation.